ar_exercise/exercise.rb

```diff
@@ -34,14 +34,21 @@ class Apartment < ActiveRecord::Base
 all_tenants = Tenant.all
 
 # get the first tenant in the DB
+first = Tenant.first
 # get all tenants older than 65
+old = Tenant.where "age>65"
```

Comment:
You're going to need a little more here. Take a look at this post: http://stackoverflow.com/questions/11317662/rails-using-greater-than-less-than-with-a-where-statement